x/shield/keeper/pool.go

```diff
@@ -24,7 +24,9 @@ func (k Keeper) GetPool(ctx sdk.Context, id uint64) (types.Pool, error) {
 	return pool, nil
 }
 
-func (k Keeper) CreatePool(ctx sdk.Context, creator sdk.AccAddress, shield sdk.Coins, deposit types.MixedCoins, sponsor string, time int64) (types.Pool, error) {
+func (k Keeper) CreatePool(ctx sdk.Context, creator sdk.AccAddress,
+	shield sdk.Coins, deposit types.MixedCoins, sponsor string,
+	sponsorAddr sdk.AccAddress, time int64) (types.Pool, error) {
```

Comment:
I think Zhaozhong mentioned we should keep function header in one line?

Reply:
It's just way too long, and he did mention it's fine to break it if it is.